stable/redmine/values.schema.json

```diff
@@ -0,0 +1,196 @@
+          "description": "Whether to deploy a mariadb server to satisfy the applications database requirements. To use an external database switch this off and configure the external database details"
+        },
+        "master": {
+          "type": "object",
+          "properties": {
+            "persistence": {
+              "type": "object",
+              "properties": {
+                "size": {
+                  "type": "string",
+                  "title": "Volume Size",
+                  "form": true,
+                  "hidden": {
+                    "condition": false,
+                    "value": "mariadb.enabled"
+                  },
+                  "render": "slider",
+                  "sliderMin": 1,
+                  "sliderMax": 100,
+                  "sliderUnit": "Gi"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "postgresql": {
+      "type": "object",
+      "title": "PostgreSQL Details",
+      "form": true,
+      "hidden": {
+        "condition": false,
+        "value": "databaseType.postgresql"
+      },
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "title": "Use a new PostgreSQL database hosted in the cluster",
+          "form": true,
+          "description": "Whether to deploy a postgresql server to satisfy the applications database requirements. To use an external database switch this off and configure the external database details"
+        },
+        "persistence": {
+          "type": "object",
+          "properties": {
+            "size": {
+              "type": "string",
+              "title": "Volume Size",
+              "form": true,
+              "hidden": {
+                "condition": false,
+                "value": "postgresql.enabled"
+              },
+              "render": "slider",
+              "sliderMin": 1,
+              "sliderMax": 100,
+              "sliderUnit": "Gi"
+            }
+          }
+        }
+      }
+    },
+    "externalDatabase": {
+      "type": "object",
+      "title": "External Database Details",
+      "description": "If MariaDB is disabled. Use this section to specify the external database details",
+      "form": true,
+      "hidden": "mariadb.enabled and postgresql.enabled",
+      "properties": {
+        "host": {
+          "type": "string",
+          "form": true,
+          "title": "Database Host"
+        },
+        "user": {
+          "type": "string",
+          "form": true,
+          "title": "Database Username"
+        },
+        "password": {
+          "type": "string",
+          "form": true,
+          "title": "Database Password"
+        },
+        "database": {
+          "type": "string",
+          "form": true,
+          "title": "Database Name"
+        },
+        "port": {
+          "type": "integer",
+          "form": true,
+          "title": "Database Port"
+        }
+      }
+    }
+  }
```

Comment:
My recommendation is to remove this parameter so we unblock this PR, making a note to add it back once the linked issue is addressed.

Reply:
Check the issue now @andresmgot 
Using "databaseType" as a string gives us even less possibilities for the JSON schema